legacy/timer.c

```
@@ -22,6 +22,7 @@
 #include <libopencm3/cm3/systick.h>
 #include <libopencm3/cm3/vector.h>
 #include <libopencm3/stm32/rcc.h>
+#include <vendor/libopencm3/include/libopencmsis/core_cm3.h>
```

Cppcheck (2.19.0)

[information] Include file

(missingIncludeSystem)